.github/actions/report-job-status/README.md

```diff
@@ -0,0 +1,31 @@
+# Send VK Teams notification on failure
+
+This action uses [report-job-status action](https://github.com/tarantool/actions/tree/master/report-job-status)
+and chooses the proper chat for the notification:
+* The team chat (Tarantool CI/CD reports) - if the job started for the stable
```

Comment:
... - if the job started on push to the `master` or release branch (`1.10`, `2.10`, etc)

Reply:
For current workflows, the triggered event also can be `scheduled`, theoretically it also can be many other: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows. 
stable -> release fixed

Reply:
Ok, got it.